From 8521bfe565eed94f3d10f86aa98c4360be205e5c Mon Sep 17 00:00:00 2001
From: Rodney Dawes <dobey.pwns@gmail.com>
Date: Fri, 10 Jun 2022 15:14:48 -0400
Subject: [PATCH 10/13] cmake: Don't build ut_preedit-string test as it was
 disabled before

Before the maliit-keyboard fork, and switch to CMake, this test was
not being built anyway, so disable it here for now as well.

Signed-off-by: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f4180d3a..b024acc1 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -455,7 +455,7 @@ if(enable-tests)
     create_test(ut_editor
             tests/unittests/ut_editor/wordengineprobe.cpp
             tests/unittests/ut_editor/wordengineprobe.h)
-    create_test(ut_preedit-string)
+    #create_test(ut_preedit-string)
     create_test(ut_text)
     create_test(ut_word-candidates
             tests/unittests/ut_word-candidates/wordengineprobe.cpp
-- 
2.30.2

